codec-http/src/main/java/io/netty/handler/codec/http/multipart/HttpPostRequestEncoder.java

```diff
@@ -92,12 +92,14 @@
         HTML5
     }
 
-    private static final Map<Pattern, String> percentEncodings = new HashMap<Pattern, String>();
+    private static final SimpleImmutableEntry[] percentEncodings;
 
     static {
-        percentEncodings.put(Pattern.compile("\\*"), "%2A");
-        percentEncodings.put(Pattern.compile("\\+"), "%20");
-        percentEncodings.put(Pattern.compile("%7E"), "~");
+        percentEncodings = new AbstractMap.SimpleImmutableEntry[] {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Unnecessary use of fully qualified name 'AbstractMap.SimpleImmutableEntry' due to existing static import 'java.util.AbstractMap.SimpleImmutableEntry' [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3AUnnecessaryFullyQualifiedName)